### DISCUSSION TITLE
[5.x] _updates table updare_id

### DISCUSSION THREAD

**Discussion by joomleb** [joomla/joomla-cms#42800] [General]
Hi guys,
PHP 8.1.x + Joomla 5.x (and also  Joomla 4.x) new installations...

Please, Why is the _updates table installed with update_id column value starting from 130 instead of 1?

**Comment by richard67** [comment]
> Hi guys, PHP 8.1.x + Joomla 5.x (and also Joomla 4.x) new installations...
> 
> Please, Why is the _updates table installed with update_id column value starting from 130 instead of 1?

@joomleb Why is that a problem for you?

**Comment by joomleb** [comment]
Hi @richard67 
Please, Where I wrote that is a problem for me ?

I made a question. 
Sure, anyway, my opinion is that if the answer is "there is no a why for it", would be a good actitude to keep new installations "as ordered/clean as possible" by alphabetically reordering them and starting from update_id 1

**Comment by richard67** [comment]
> Hi @richard67 Please, Where I wrote that is a problem for me ?
> 
> I made a question. Sure, anyway, my opinion is that if the answer is "there is no a why for it", would be a good actitude to keep new installations "as ordered/clean as possible" by alphabetically reordering them and starting from update_id 1

@joomleb It's just that your question doesn't make sense. As you can see here, the column is an auto increment column: https://github.com/joomla/joomla-cms/blob/5.0-dev/installation/sql/mysql/base.sql#L852 .

The value of that column does not have any meaning for the user, and so it doesn't matter if it starts from 1 or whatever.

As soon as there was some update found and you clear the updates table by looking again for updates, the value will change.
